frontend/appflowy_tauri/src/appflowy_app/components/document/CodeBlock/SelectLanguage.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useCallback, useContext } from 'react';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { updateNodeDataThunk } from '$app_reducers/document/async-actions';
+import { useAppDispatch } from '$app/stores/store';
+import { DocumentControllerContext } from '$app/stores/effects/document/document_controller';
+import { supportLanguage } from '$app/constants/document/code';
+
+function SelectLanguage({ id, language }: { id: string; language: string }) {
```

Comment:
the parameter of this function looks a little confusing.

can it be equal to `function({id: String, language: string})`?